src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -2798,3 +2801,239 @@ export function getMolecularProfileSamplesSet(
+    );
+    const tierFilterIsActive = tierFilterActive(
+        geneFilterQuery.tiersBooleanMap,
+        geneFilterQuery.includeUnknownTier
+    );
+    const statusFilterIsActive = statusFilterActive(
+        geneFilterQuery.includeGermline,
+        geneFilterQuery.includeSomatic,
+        geneFilterQuery.includeUnknownStatus
+    );
+    // const annotationFilterIsActive = annotationFilterActive(true, true, true);
+    const isMutationType =
+        chartType === ChartTypeEnum.MUTATED_GENES_TABLE ||
+        chartType === ChartTypeEnum.FUSION_GENES_TABLE;
+    if (
+        !annotationFilterIsActive &&
+        !tierFilterIsActive &&
+        (!statusFilterIsActive || !isMutationType)
+    )
+        return null;
+
+    const driverFilterTextElements: string[] = [];
+    if (annotationFilterIsActive) {
+        geneFilterQuery.includeDriver &&
+            driverFilterTextElements.push('driver');
+        geneFilterQuery.includeVUS &&
+            driverFilterTextElements.push('passenger');
+        geneFilterQuery.includeUnknownOncogenicity &&
+            driverFilterTextElements.push('unknown');
+    }
+
+    const statusFilterTextElements: string[] = [];
+    if (statusFilterIsActive && isMutationType) {
+        geneFilterQuery.includeGermline &&
+            statusFilterTextElements.push('germline');
+        geneFilterQuery.includeSomatic &&
+            statusFilterTextElements.push('somatic');
+        geneFilterQuery.includeUnknownStatus &&
+            statusFilterTextElements.push('unknown');
+    }
+
+    const tierNames = tierFilterIsActive
+        ? _(geneFilterQuery.tiersBooleanMap)
+              .pickBy()
+              .keys()
+              .value()
+        : [];
+    if (tierFilterActive && geneFilterQuery.includeUnknownTier)
+        tierNames.push('unknown');
+
+    let driverFilterText = '';
+    if (driverFilterTextElements.length === 1)
+        driverFilterText = driverFilterTextElements[0];
+    else if (driverFilterTextElements.length > 1)
+        driverFilterText =
+            driverFilterTextElements.slice(0, -1).join(', ') +
+            ' or ' +
+            driverFilterTextElements.slice(-1);
+
+    let statusFilterText = '';
+    if (statusFilterTextElements.length === 1)
+        statusFilterText = statusFilterTextElements[0];
+    else if (statusFilterTextElements.length > 1)
+        statusFilterText =
+            statusFilterTextElements.slice(0, -1).join(', ') +
+            ' or ' +
+            statusFilterTextElements.slice(-1);
+
+    let tierFilterText = '';
+    if (tierNames.length === 1) tierFilterText = tierNames[0];
+    else if (tierNames.length > 1)
+        tierFilterText =
+            tierNames.slice(0, -1).join(', ') + ' or ' + tierNames.slice(-1);
+
+    return (
+        <div data-test={'groupedGeneFilterIcons'} className={styles.content}>
+            {driverFilterText && (
+                <div
+                    style={{
+                        display: 'flex',
```

Comment:
we should maybe check whether these inline styles are necessary.  and if so, whehter they can be applied using a class